config/final.yml

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
Looks like this is taken from cf-release. I think we should remove this in subsequent commits (if we don't intend to make final release of this job or adjust to point to our bucket), because now it points to the same bucket as CF-release.